.circleci/config.yml

```diff
@@ -229,6 +229,9 @@ jobs:
           DJANGO_SETTINGS_MODULE: settings
           DJANGO_CONFIGURATION: Test
           DJANGO_SECRET_KEY: ThisIsAnExampleKeyForTestPurposeOnly
+          LTI_1_BASE_URL: http://localhost:8060/lti/videos/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}
+          LTI_1_OAUTH_CONSUMER_KEY: InsecureOauthConsumerKey
+          LTI_1_SHARED_SECRET: InsecureSharedSecret
```

Comment:
I think we should use the LTI testing tool as example in the sandbox : https://lti.tools/saltire/
```
url: https://lti.tools/saltire/tp
consumer key: your choice
shared secret: secret
```

Reply:
About the LTI_1, I thought of what you said that you wanted to convey the fact that it allow more than one... but in the case of this sandbox setting, only one is possible via environment variables. So I'm not a big fan of these names... so since we will be pointing to the LTI test servi ce, I propose to name them: 
`LTI_TEST_BASE_URL`, `LTI_TEST_OAUTH_CONSUMER_KEY` and `LTI_TEST_SHARED_SECRET`.

Reply:
Shall I use a regex_url for https://lti.tools/saltire/tp ?

Reply:
well, there is no variable uuid in it :man_shrugging: so I would say no.